kvdb-rocksdb/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "kvdb-rocksdb"
-version = "0.1.4"
+version = "0.2.4"
```

Comment:
why a major bump? ethereum-types is a dev-dependency

Reply:
mybad